ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -1321,6 +1323,7 @@ export class ParticipantListComponent implements OnInit {
     } else {
       this.selectedColumns[ parent ].push(column);
     }
+    console.log(this.selectedColumns);
```

Comment:
Do we need this in prod code, or was this something we needed along the way during development?

Reply:
omg I missed this, thanks!